platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformSchedulers.java

```diff
@@ -113,6 +140,7 @@ public static PlatformSchedulers create(
                         .withOnRamp(hashingObjectCounter)
                         .withExternalBackPressure(true)
                         .withMetricsBuilder(model.metricsBuilder().withUnhandledTaskMetricEnabled(true))
+                        .withHyperlink(platformCoreHyperlink(EventHasher.class))
```

Comment:
A note to reviewers: this will become 100% automated once we construct schedulers automatically inside the proxy `ComponentWiring` classes (this is a planned feature).